From b5c55559b4a078e3c26033196f421748b56f27c8 Mon Sep 17 00:00:00 2001
From: Johan Klokkhammer Helsing <johan.helsing@qt.io>
Date: Fri, 13 Sep 2019 12:52:15 +0200
Subject: [PATCH] Don't try to inject non-existent generated headers for
 xdg-shell v5

We use a hand edited version for xdg-shell-v5, which is added through
src/plugins/shellintegration/xdg-shell-v5/pregenerated/xdg-shell-v5.pri so we
shouldn't need to add it through sync.profile.

MJ: Added (q)wayland-server-xdg-shell-unstable-v5.h as well.

Fixes: QTBUG-78210
Change-Id: Ida60f98c950b74132bd526cf44a4bf667cdce730
Reviewed-by: Paul Olav Tvete <paul.tvete@qt.io>
---
 sync.profile | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/sync.profile b/sync.profile
index 3d0ba53..7eaad4d 100644
--- a/sync.profile
+++ b/sync.profile
@@ -44,10 +44,6 @@
         "^wayland-xdg-shell-client-protocol.h",
         "^wayland-xdg-decoration-unstable-v1-client-protocol.h",
     ],
-    "$basedir/src/plugins/shellintegration/xdg-shell-v5" => [
-        "^qwayland-xdg-shell-unstable-v5.h",
-        "^wayland-xdg-shell-unstable-v5-client-protocol.h",
-    ],
     "$basedir/src/plugins/shellintegration/xdg-shell-v6" => [
         "^qwayland-xdg-shell-unstable-v6.h",
         "^wayland-xdg-shell-unstable-v6-client-protocol.h",
@@ -62,7 +58,6 @@
         "^qwayland-server-text-input-unstable-v2.h",
         "^qwayland-server-touch-extension.h",
         "^qwayland-server-xdg-decoration-unstable-v1.h",
-        "^qwayland-server-xdg-shell-unstable-v5.h",
         "^qwayland-server-xdg-shell-unstable-v6.h",
         "^qwayland-server-xdg-shell.h",
         "^wayland-hardware-integration-server-protocol.h",
@@ -75,7 +70,6 @@
         "^wayland-wayland-server-protocol.h",
         "^wayland-xdg-decoration-unstable-v1-server-protocol.h",
         "^wayland-xdg-shell-server-protocol.h",
-        "^wayland-xdg-shell-unstable-v5-server-protocol.h",
         "^wayland-xdg-shell-unstable-v6-server-protocol.h",
     ],
 );
-- 
2.17.1

